app/views/tags/show.haml

```diff
@@ -1,24 +1,41 @@
 - content_for :breadcrumbs do
   = link_to "Tags", tags_path
   &raquo;
-  - if @tag.type
-    = @tag.type.pluralize.titlecase
-    &raquo;
+  = @tag.type.pluralize.titlecase
+  &raquo;
   %b= @tag.name
 
-- content_for :edit_box do
-  - if @tag.editable_by?(current_user)
-    = link_to edit_tag_path(@tag) do
-      .link-box.blue
-        = image_tag "icons/pencil.png"
-        Edit
-    = link_to tag_path(@tag), method: :delete, data: { confirm: 'Are you sure you want to delete this tag?' } do
-      .link-box.red x Delete
+%table
+  %thead
+    %tr
+      %th{colspan: 2}
+        #{@tag.type}: #{@tag.name}
+        - if @tag.editable_by?(current_user)
+          = link_to edit_tag_path(@tag) do
+            .link-box.blue
+              = image_tag "icons/pencil.png"
+              Edit
+          = link_to tag_path(@tag), method: :delete, data: { confirm: 'Are you sure you want to delete this tag?' } do
+            .link-box.red x Delete
+  %tbody
+    - if @tag.is_a?(Setting) && @tag.canons.present?
+      %tr
+        %th.centered.sub= 'Canon'.pluralize(@tag.canons.count)
+        %td{class: cycle('even', 'odd')}= @tag.canons.map { |c| link_to(c.name, tag_path(c))}.join(', ').html_safe
+    - if @tag.description.present?
+      %tr
+        %th.centered.sub.width-150.vtop Description
+        %td.written-content{class: cycle('even', 'odd')}= sanitize_written_content(@tag.description).html_safe
+    - unless [@posts, @galleries, @characters, (@tag.settings rescue [])].any?(&:present?)
```

Comment:
I hear inline rescue is a bad thing to use but I'm not sure what to replace it with (`@tag.respond_to?(:settings) ? @tag.settings : []`?)